contrib/seldon/Makefile

```diff
@@ -10,17 +10,35 @@
 
 seldon-core-operator/base: clean-kustomize
 	mkdir -p seldon-core-operator/base
-	cd seldon-core-operator/base && helm template -f ../../values.yaml seldon-core seldon-core-operator --repo https://storage.googleapis.com/seldon-charts --namespace kubeflow --version 1.6.0  > resources.yaml
+	cd seldon-core-operator/base && helm template -f ../../values.yaml seldon-core seldon-core-operator --repo https://storage.googleapis.com/seldon-charts --namespace kubeflow --version 1.15.0  > resources.yaml
 	#kfctl strangeness means these variables don't always get resolved see https://github.com/kubeflow/manifests/issues/1660
 	#sed -i 's#cert-manager.io/inject-ca-from:.*#cert-manager.io/inject-ca-from: $$(CERTIFICATE_NAMESPACE)/$$(CERTIFICATE_NAME)#g' seldon-core-operator/base/resources.yaml
 	sed -i 's#cert-manager.io/inject-ca-from:.*#cert-manager.io/inject-ca-from: kubeflow/seldon-serving-cert#g' seldon-core-operator/base/resources.yaml
-	sed -i "s#'seldon-webhook-service.kubeflow.svc.cluster.local'#"'$$(SERVICE_NAME).$$(SERVICE_NAMESPACE).svc.cluster.local#' seldon-core-operator/base/resources.yaml
-	sed -i "s#commonName: 'seldon-webhook-service.kubeflow.svc'#"'commonName: $$(SERVICE_NAME).$$(CERTIFICATE_NAMESPACE).svc#' seldon-core-operator/base/resources.yaml
-	sed -i "s#'seldon-webhook-service.kubeflow.svc'#"'$$(SERVICE_NAME).$$(SERVICE_NAMESPACE).svc#' seldon-core-operator/base/resources.yaml
+	sed -i "s#'seldon-webhook-service.kubeflow.svc.cluster.local'#"'seldon-webhook-service.$$(SERVICE_NAMESPACE).svc.cluster.local#' seldon-core-operator/base/resources.yaml
+	sed -i "s#commonName: 'seldon-webhook-service.kubeflow.svc'#"'commonName: seldon-webhook-service.$$(CERTIFICATE_NAMESPACE).svc#' seldon-core-operator/base/resources.yaml
+	sed -i "s#'seldon-webhook-service.kubeflow.svc'#"'seldon-webhook-service.$$(SERVICE_NAMESPACE).svc#' seldon-core-operator/base/resources.yaml
 	sed -i 's#kubeflow/kubeflow-gateway#$$(SERVICE_NAMESPACE)/kubeflow-gateway#' seldon-core-operator/base/resources.yaml
 	cp kustomization.tpl seldon-core-operator/base/kustomization.yaml
 	cp kustomizeconfig.tpl seldon-core-operator/base/kustomizeconfig.yaml
 
 .PHONY:clean-kustomize
 clean-kustomize:
 	rm -rf seldon-core-operator/base
+
+test:
```

Comment:
@kimwnasptd I think this should be sufficient to meet the test requirement for contrib, do we want to encourage to add the test into github actions workflow as well?

Reply:
I'd propose that yes, let's have a very minimal test in place for the GH Actions. This will allow us to run these tests periodically to easily smoke test these components

Reply:
@axsaucedo could you add a simple test workflow `.github/workflows/contrib_seldon_core.yaml` and define a minimal test case?

It can be something as simple as just verifying that the Pods of seldon core become ready. You can use as reference the rest of the apps that we are testing https://github.com/kubeflow/manifests/blob/master/.github/workflows/jwa_kind_test.yaml

If you want to test slightly more advanced cases, like creating a CR and testing if it also becomes Ready then that would also be more that welcome! But as a firs step testing that the core Pods are up should be enough

Reply:
Actually, just running this `make test` inside of the GH Action should suffice

Reply:
Ok sounds good, let me add that